### PR TITLE
Add warning to ensure artifact name matches mender.conf

### PR DESCRIPTION
**Pull request by drewmoseley**
Signed-off-by: Drew Moseley <drew.moseley@northern.tech>

### CODE REVIEW

File: 04.Artifacts/06.Signing-and-verification/docs.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ and if you are signing an existing Mender Artifact (`.mender`).
 mender-artifact write rootfs-image -t beaglebone -n mender-1.0.1 -u core-image-base-beaglebone.ext4 -k private.key -o artifact-signed.mender
 ```
 
+! Make sure the artifact name specified by the `-n` parameter in the above command matches the value specified in the `mender.conf` file in your file system image.
+
```
Comment (mirzak) [inline 04.Artifacts/06.Signing-and-verification/docs.md]: It is not part of the `mender.conf` file, its `/etc/mender/artifact_info` right?

Comment (drewmoseley) [reply 04.Artifacts/06.Signing-and-verification/docs.md]: Yes, indeed.  I'll reword it.

Comment (estenberg) [reply 04.Artifacts/06.Signing-and-verification/docs.md]: Would capitalize Artifact.

Comment (drewmoseley) [reply 04.Artifacts/06.Signing-and-verification/docs.md]: Fixed

**Pull request merged**